File: sdkwork-app-sdk-typescript/src/types/exchange-record-query-form.ts
/** 兑换记录查询请求 */
export interface ExchangeRecordQueryForm {
  /** 页码 */
  pageNum?: number;
  /** 每页大小 */
  pageSize?: number;
  /** 排序字段 */
  sortField?: string;
  /** 排序方向 */
  sortDirection?: string;
  /** 兑换状态: pending/shipped/completed/cancelled */
  status?: string;
  size?: number;
  page?: number;
}

File: sdkwork-app-sdk-go/types/exchange_record_query_form.go
package types

// 兑换记录查询请求
type ExchangeRecordQueryForm struct {
	PageNum int `json:"pageNum"`
	PageSize int `json:"pageSize"`
	SortField string `json:"sortField"`
	SortDirection string `json:"sortDirection"`
	Status string `json:"status"`
	Size int `json:"size"`
	Page int `json:"page"`
}
